db/migrate/20250729132443_add_notices_to_order_details.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddNoticesToOrderDetails < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
Should we add a rollback here?

Reply:
Do you mean to make this migration revertable? because it already is